drivers/scheduler/rke/rke.go

```diff
@@ -69,61 +61,63 @@ func (r *Rancher) Init(scheduleOpts scheduler.InitOptions) error {
 	}
 	return nil
 }
-*/
 
 // GetRancherClusterParametersValue returns the rancher token, endpoint, secret key, access key
 func (r *Rancher) GetRancherClusterParametersValue() (*RancherClusterParameters, error) {
-	var rkeData map[string]interface{}
 	var rkeParameters RancherClusterParameters
+	var rkeToken string
 	// TODO Rancher URL for cloud cluster will not be fetched from master node IP
 	masterNodeName := node.GetMasterNodes()[0].Name
 	endpoint := "https://" + masterNodeName + "/v3"
-	rancherURL := "https://" + masterNodeName + "/v3-public/localProviders/local?action=login"
-	//TODO: Get the values from config map after https://github.com/portworx/torpedo/pull/1517 is merged
-	username := ""
-	password := ""
-	tr := &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
-	}
-	client := &http.Client{Transport: tr}
-	body := strings.NewReader(fmt.Sprintf(`{"username":"%s","password":"%s"}`, username, password))
-	req, err := http.NewRequest("POST", rancherURL, body)
-	if err != nil {
-		log.Errorf("Failed to create rancher POST request:", err)
-		return nil, err
-	}
-	req.Header.Add("Content-Type", "application/json")
-	resp, err := client.Do(req)
-	if err != nil {
-		log.Errorf("Failed to send request:", err)
-		return nil, err
-	}
-	defer resp.Body.Close()
-	respBody, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		log.Errorf("Failed to read response:", err)
-		return nil, err
-	}
-	response := string(respBody)
-	err = json.Unmarshal([]byte(response), &rkeData)
-	if err != nil {
-		log.Errorf("Error:", err)
-		return nil, err
-	}
 	rkeParameters.Endpoint = endpoint
-	rkeToken, res := rkeData["token"].(string)
-	if res != true {
-		return nil, fmt.Errorf("token in rkeData is not of string type")
-	}
-	if len(strings.Split(rkeToken, ":")) != 2 {
-		return nil, fmt.Errorf("invalid RKE token value")
-	}
+	rkeToken = os.Getenv("SOURCE_RKE_TOKEN")
+	gomega.Expect(rkeToken).NotTo(gomega.Equal(""),
+		"RKE_TOKEN Environment variable should not be empty")
 	rkeParameters.Token = rkeToken
 	rkeParameters.AccessKey = strings.Split(rkeToken, ":")[0]
 	rkeParameters.SecretKey = strings.Split(rkeToken, ":")[1]
 	return &rkeParameters, nil
 }
 
+// UpdateClient updates the rancher client based on the current cluster context
+func (r *Rancher) UpdateClient(clusterName string) error {
```

Comment:
Pls use "Rancher" string in function name as well.

Reply:
done